### PR TITLE
Improve rendering of buttons in the toolbar in HCM (bug 1845515)

### PR DESCRIPTION
**Pull request by calixteman**
With this patch, in HCM, make a button visible when it's active and add an outline around toggled buttons when they're hovered.

### CODE REVIEW

File: web/viewer.css [hunk 1]
```diff
@@ -322,7 +325,6 @@ body {
 :is(.toolbar, .editorParamsToolbar, .findbar, #sidebarContainer)
   :is(input, button, select),
 .secondaryToolbar :is(input, button, a, select) {
-  outline: none;
```
Comment (Snuffleupagus) [inline web/viewer.css:325]: This affects much more than HCM-mode, note how e.g. the zoom-dropdown now has an unexpected/inconsistent outline when tabbing through the toolbar buttons.
The same "issue" also applies to the entries in the various editor-toolbars.

File: web/viewer.css [hunk 2]
```diff
@@ -181,6 +182,8 @@
     --toolbar-icon-opacity: 1;
     --toolbar-icon-bg-color: ButtonText;
     --toolbar-icon-hover-bg-color: ButtonFace;
+    --toggled-hover-active-btn-color: ButtonText;
+    --toggled-hover-btn-outline: 3px solid ButtonBorder;
```
Comment (Snuffleupagus) [inline web/viewer.css:186]: Will the buttons still "fit" vertically inside of the toolbar with such a thick outline?

**Review by Snuffleupagus — CHANGES_REQUESTED** [review]
See inline comments, and please make sure to *carefully* test the viewer since these sort of changes can very easily "break" a bunch of (unintended) things.

File: web/viewer.css (hunk 2 above)
Comment (calixteman) [reply web/viewer.css]: With 2px it fits.

**Review by Snuffleupagus — APPROVED** [review]
r=me, since this is now limited to only HCM mode; thank you!

(Possibly we should re-factor all button-colors/outlines/hover/... but that should happen separately, such that everything is covered and changed all at once if so.)

**Pull request merged**